README.md

```diff
@@ -120,6 +120,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
 * [Brunch](https://github.com/brunch/brunch) - Fast front-end web app build tool with simple declarative config.
 * [Parcel](https://github.com/parcel-bundler/parcel) - Blazing fast, zero configuration web application bundler.
 * [Microbundle](https://github.com/developit/microbundle) - Zero-configuration bundler for tiny modules.
+* [FuseBox](https://github.com/fuse-box/fuse-box) - A Zero-configuration bundler which is used programmatically.
```

Comment:
Can you please update this line with their own description?

Reply:
Done